app/_locales/el/messages.json

```diff
@@ -5269,12 +5640,242 @@
   "settingsSubHeadingSignaturesAndTransactions": {
     "message": "Αιτήματα υπογραφών και συναλλαγών"
   },
+  "shieldClaim": {
+    "message": "Υποβολή αξίωσης"
+  },
+  "shieldClaimDescription": {
+    "message": "Περιγραφή αιτήματος"
+  },
+  "shieldClaimDetails": {
+    "message": "Μπορείτε να υποβάλετε μία μόνο αξίωση ανά περιστατικό εντός 21 ημερών από το περιστατικό. Δείτε την πλήρη πολιτική κάλυψης $1.",
+    "description": "The $1 is a link to the full coverage policy"
+  },
+  "shieldClaimEmail": {
+    "message": "Email"
+  },
+  "shieldClaimEmailHelpText": {
+    "message": "Θα χρησιμοποιήσουμε αυτό το email για να σας κρατάμε ενήμερους με ενημερώσεις."
+  },
+  "shieldClaimFileUploader": {
+    "message": "Μεταφόρτωση εικόνας"
+  },
+  "shieldClaimFileUploaderHelpText": {
+    "message": "Αν έχετε τυχόν πρόσθετα αποδεικτικά στοιχεία, ανεβάστε τα εδώ."
+  },
+  "shieldClaimImpactedTxHash": {
+    "message": "Κωδικός (hash) της συναλλαγής που επηρεάστηκε"
+  },
+  "shieldClaimImpactedTxHashHelpText": {
+    "message": "Γνωστό επίσης ως TXID, hash ID ή αναγνωριστικό συναλλαγής."
+  },
+  "shieldClaimImpactedTxHashHelpTextLink": {
+    "message": "Χρειάζεστε βοήθεια για να το βρείτε;"
+  },
+  "shieldClaimImpactedWalletAddress": {
+    "message": "Διεύθυνση πορτοφολιού που επηρεάστηκε"
+  },
+  "shieldClaimInvalidEmail": {
+    "message": "Παρακαλούμε εισάγετε μια έγκυρη διεύθυνση email"
+  },
+  "shieldClaimReimbursementWalletAddress": {
+    "message": "Διεύθυνση πορτοφολιού για επιστροφή χρημάτων"
+  },
+  "shieldClaimReimbursementWalletAddressHelpText": {
+    "message": "Bεβαιωθείτε ότι η διεύθυνση πορτοφολιού δεν έχει παραβιαστεί."
+  },
+  "shieldClaimSubmit": {
+    "message": "Υποβολή αξίωσης"
+  },
+  "shieldConfirmMembership": {
+    "message": "Επιβεβαίωση συνδρομής"
+  },
+  "shieldEntryModalAssetCoverage": {
+    "message": "Κάλυψη περιουσιακών στοιχείων έως $1 ανά συναλλαγή",
+    "description": "The $1 is the amount of asset coverage per transaction"
+  },
+  "shieldEntryModalDescription": {
+    "message": "Ανανεώνεται με $1/μήνα, ακύρωση οποιαδήποτε στιγμή. $2",
+    "description": "The $1 is the price of the shield and $2 is a link to the learn more page"
+  },
   "shieldEntryModalGetStarted": {
     "message": "Ξεκινήστε"
   },
+  "shieldEntryModalProtection": {
+    "message": "Προστασία σε όλους τους λογαριασμούς σας"
+  },
   "shieldEntryModalSkip": {
     "message": "Παράλειψη"
   },
+  "shieldEntryModalSubtitle": {
+    "message": "Προστασία συναλλαγών, δωρεάν για 14 ημέρες."
+  },
+  "shieldEntryModalSupport": {
+    "message": "Υποστήριξη πελατών με προτεραιότητα"
+  },
+  "shieldEstimatedChangesMonthlyTooltip": {
+    "message": "Εξουσιοδοτήστε έως και $$1 για ολόκληρο το έτος. Θα χρεώνεστε $$2 κάθε μήνα, όχι ολόκληρο το ποσό τώρα."
```

Comment:
Hi @i18nlaurel , I see we have double $ character in several places. Is it expected or is it an error?